### ISSUE TITLE
NVIDIA GTX TITAN X (GM200) doesn't work in any game with dxvk 2.0+

### ISSUE THREAD

**Issue by keybreak** [doitsujin/dxvk#3790]
Arch Linux / Nvidia proprietary drivers.
My problem is that i can not launch **any** game using [dxvk 2.0+](https://github.com/doitsujin/dxvk/wiki/Driver-support), despite it seems to meet all the outlined requirements. (it made me sad when 2.0+ came out like that...but i had no idea that my GPU would be affected, it's far from being dinosaur).

That also means i can't use wine / proton fixes that come out with wine newer wine versions higher than 7.1...
Perhaps my particular GPU requires some extra-special treatment or fixes that was forgotten during bumping requirements of dxvk code?

### Software information
Any game that is not DirectX 12, which is not scope of DXVK.

### System information
- GPU: [NVIDIA GeForce GTX TITAN X (GM200)](https://www.techpowerup.com/gpu-specs/geforce-gtx-titan-x.c2632)
- Driver: 545.29.06
- Wine version: any 7.1+
- DXVK version: any 2.0+

**Comment by K0bin** [comment]
That GPU is a Maxwell 2.0 card, so it's still supported and your driver looks reasonably recent.

Please provide logs, there's no way we can help you without more information to go by.

**Comment by keybreak** [comment]
@K0bin 
Which kind of logs would be helpful to debug?

**Comment by Riesi** [comment]
Do things run with dxvk disabled?
Those errors dont look like dxvk's fault. Its rather wine dying immediately.

**Comment by keybreak** [comment]
@Riesi 
Huh...You're right. It actually dies same way with dxvk disabled as well, sorry for false alarm then...
Although, i have to admit it looks weird, because it perfectly aligns with new requirements, no wine / proton version higher than 7.1+ works for me....and i've tried all kinds of versions. With 7.1- and dxvk v2.0- everything works... :thinking:

**Comment by K0bin** [comment]
> Which kind of logs would be helpful to debug?

DXVK logs. But it doesn't look like DXVK even gets loaded.

**Comment by Riesi** [comment]
Are you reusing the same prefix? Maybe upgrading broke somehow.
Try a fresh one by simply changing the path under "Game options"-> "Wine prefix"

**Comment by keybreak** [comment]
@Riesi 
> Are you reusing the same prefix?

Of course, i would get insane if i'd need to create wineprefix every time wine hits major version!
I've thought...

But  you were absolutely right, it works perfectly with freshly squeezed prefix, 8.25-wine-ge + dxvk 2.3!
Thank you so much for your help! :clown_face:  :frog: 

Now i'll have to figure out if i need to manually recreate 60 prefixes or there is a way to unf*ck process of updating wine 6 to 7 or 8 without breaking it... :thinking:

**Comment by Riesi** [comment]
Great that things seem to work now! :frog: 

I am not sure if there is an extra command to regenerate only parts of a prefix, since wine should do the right thing on version changes already. But that didnt work for you it seems :(
I hope you can figure something out!

**Comment by keybreak** [comment]
@Riesi 
Most of my prefixes were created back in 5.x wine era...
So far i've tried to do gradual upgrade on one of prefixes, 5 -> 6 -> 7 -> 8, with usual `winecfg` method...seems to work although during specifically 6 ->7 step i've noticed that in terminal:

```
warning: Your version of wine 5.12-13990-g10209e2e433 is no longer supported upstream. You should upgrade to 6.x
```

I wonder where exactly does it gets this version string from, given it's already upgraded to 6 by then. :thinking:
